portable/NetworkInterface/ThirdParty/MSP432/NetworkInterface.c

```diff
@@ -271,8 +285,13 @@ bool setupEMAC()
     SysCtlPeripheralReset( SYSCTL_PERIPH_EMAC0 );
     SysCtlPeripheralReset( SYSCTL_PERIPH_EPHY0 );
 
-    while( !SysCtlPeripheralReady( SYSCTL_PERIPH_EMAC0 ) )
+    /* The peripheral should always start, but there is a wait with timeout here to prevent an infinite loop*/
+    ul32Loop = 0;
+
+    while( !SysCtlPeripheralReady( SYSCTL_PERIPH_EMAC0 ) && ( ul32Loop < ETH_STARTUP_TIMEOUT ) )
```

Comment:
If you pass line 296 with `ul32Loop > ETH_STARTUP_TIMEOUT`, this code continues as if everything is fine when the EMAC never became ready.  In this case it should not continue with the initialization and should return `false`.

Reply:
Done, this has been updated.